builtin/staker/staker.go

```diff
@@ -645,8 +656,15 @@ func (s *Staker) validateStakeIncrease(validator thor.Address, validation *valid
 	if err != nil {
 		return err
 	}
-	if valNextPeriodTVL+aggNextPeriodTVL+amount > MaxStakeVET {
-		return NewReverts("stake is out of range")
+
+	if valNextPeriodTVL > MaxStakeVET || aggNextPeriodTVL > MaxStakeVET || amount > MaxStakeVET {
```

Comment:
Isn't this the same as before?

Reply:
Amount check is required, nextPeroidTVL is redundant? Since every increase is checked?

Reply:
Yeah, left it there does not harm anything.